lib/streams.gi

```diff
@@ -1102,6 +1124,25 @@ function( str, append )
     fi;
 end );
 
+InstallMethod( OutputTextFileRaw,
+    "output raw text stream from file",
+    [ IsString,
+      IsBool ],
+function( str, append )
+    local   fid;
+    str := UserHomeExpand(str);
+
+    fid := OUTPUT_TEXT_FILE_RAW( str, append );
```

Comment:
So, I don't want to object to the merging of this PR, but I still wonder this: There's a lot of code duplication in this PR. But I wonder, couldn't we just add an extra argument `BOOL useRawMode` or so to `INPUT_TEXT_FILE` and `OUTPUT_TEXT_FILE` ? And then add an optional second/third argument to `InputTextFile` and `OutputTextFile`?  (granted, the latter gets a bit messy; I wish GAP had keyword arguments; abusing options for that isn't so nice because of unintended consequences due to their recursive nature)

Reply:
for some reason I thought some packages were calling these methods directly, but now I don't know why I thought that as I can't find any packages which do. I'll merge.